tests/etc/full.ini

```diff
@@ -57,6 +57,9 @@ functions = 'tests/etc/functions_good.py'
 ; name the service providers you will be using.
 launch    = log, file, tests.acme.foobar, tests/acme/foobar.py
 
+; Publish the current mqttwarn version to the topic (retained)
+publishversion = mqttwarn/version
+
```

Comment:
Can you add this configuration option to `mqttwarn/examples/basic/mqttwarn.ini` instead?

Reply:
Can you add a corresponding section about that option to the documentation in `HANDBOOK.md`?

Reply:
Thanks, wasn't sure where I was to add it, and the only config I saw that was "full" was in this example....